src/javascript/app/pages/bottom/graph.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import { Skeleton } from '@deriv-com/quill-ui';
+
+const Graph = ({ renderGraph }) => {
+    const [isMounted, setMounted] = useState(false);
+
+    useEffect(() => {
+        const mountTimer = setTimeout(() => {
+            setMounted(true);
+            const renderTimer = setTimeout(() => {
+                const cleanupRender = renderGraph();
+
+                return cleanupRender;
+            }, 1000);
+
+            return () => clearTimeout(renderTimer);
+        }, 1000);
+
+        return () => clearTimeout(mountTimer);
+    }, [renderGraph]);
+
+    if (isMounted) {
+        return (
+            <div id='tab_graph' className='chart-section'>
+                <p className='error-msg' id='chart-error' />
+                <div id='trade_live_chart'>
+                    <div id='webtrader_chart' />
+                </div>
+            </div>
+        );
+    }
+
+    return <Skeleton.Square rounded fullWidth height={300} />;
```

Comment:
```suggestion
    return <Skeleton.Square rounded height={300} />;
```
fullWidth is true by default :)